ReactNativeClient/lib/models/Setting.js

```diff
@@ -316,6 +316,17 @@ class Setting extends BaseModel {
 			},
 			'folders.sortOrder.reverse': { value: false, type: Setting.TYPE_BOOL, public: true, label: () => _('Reverse sort order'), appTypes: ['cli'] },
 			trackLocation: { value: true, type: Setting.TYPE_BOOL, section: 'note', public: true, label: () => _('Save geo-location with notes') },
+
+			editorBeta: {
+				value: false,
+				type: Setting.TYPE_BOOL,
+				section: 'note',
+				public: true,
+				appTypes: ['mobile'],
+				label: () => _('Opt-in to the editor beta'),
+				description: () => _('This beta adds list continuation, Markdown preview, and Markdown shortcuts. If you find bugs, please report them in the Discourse forum.'),
```

Comment:
Please remove the `_()` markers as we don't need to translate these strings, since I assume we'll take the editor out of beta soon.